Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.    vm_nic_pairs = []
</pre><p>You seem to have a few repeating patterns of inline code here. Could they be a function?</p><p>Then just call each func + params at the bottom of the cell?</p><p>You can pass functions as parameters - e.g.</p><pre class="ql-syntax" spellcheck="false">def get_associated_item(data, column, lookup_func):
   pairs = []
   for assoc_item in data[column]:
      pairs.append(assoc_item, lookup_func(assoc_item)
   return pairs
</pre><p>Btw you also create dicts for each of these but I don't see where these are used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>I think they could be a function, but since I'm naming the lists of tuples and dicts within each for loop, it seems better to do them outside of a function, but let me know if there's an alternative to that!</p><p> </p><p>I use some of the dicts in the next cell when I'm adding edges. I could remove the others but they seem like they might be useful if other functionality were to be built?</p>